styles.css

```diff
@@ -282,3 +281,183 @@ body {
   padding-left: 1px;
   margin-left: 1px;
 }
+
+@media only screen and (min-width: 760px) {
```

Comment:
- [ ] As previously requested, let us implement this feature. min-width should be between 992px - 1100px.

> Great job using 970px for your desktop version since it's in the range of 992px - 1100px, however, It seems that your mobile version was broken while working on the desktop version,